src/validation/custom.validation.ts

```diff
@@ -169,6 +170,26 @@ export const checkAllDateFields = (dayStr: string = "", monthStr: string = "", y
   }
 };
 
+export const checkDateForFilingDate = (dayStr: string = "", monthStr: string = "", yearStr: string = ""): boolean => {
```

Comment:
I guess you followed the pattern of the other function you copied for the function name here, but would it be simpler (and correct) to just name it `checkFilingDate`?

Reply:
updated